tox.ini

```diff
@@ -72,6 +72,7 @@ deps =
     py{310,311}: psycopg2-binary==2.9.5
     py{310,311}: pyzmq==25.0.0
     dnspython1: dnspython<2
+    asyncio: aiohttp
```

Comment:
is aiohttp is only required by tests? I suppose that yes.

Reply:
Yes, it's just a testing thing.